featurebyte/query_graph/node/generic.py

```diff
@@ -685,14 +686,24 @@ class GroupByNodeParameters(BaseGroupbyParameters):
 
     windows: List[Optional[str]]
     timestamp: InColumnStr
-    blind_spot: int
-    time_modulo_frequency: int
-    frequency: int
     names: List[OutColumnStr]
+    feature_job_setting: FeatureJobSetting
```

Comment:
Refactor this part by adding a nested `FeatureJobSetting` object. By doing this, it makes us easier to extend for other feature job setting format. For example:
```
feature_job_setting: Union[FeatureJobSetting, CronJobFeatureJobSetting]
```